udata/features/territories/commands.py

```diff
@@ -51,34 +45,66 @@ def load_logos(filename):
 
 
 @m.command
-def collect_references_files():
-    """Retrieve locally CSV files in use for dynamic territories' resources."""
-    REFERENCES_PATH = references.root
-    if not os.path.exists(REFERENCES_PATH):
-        os.makedirs(REFERENCES_PATH)
-    if current_app.config.get('ACTIVATE_TERRITORIES'):
-        # TERRITORY_DATASETS is a dict of dicts and we want all values
-        # that are resource based to collect related files.
-        territory_classes = [
-            territory_class
-            for territory_dict in TERRITORY_DATASETS.values()
-            for territory_class in territory_dict.values()
-            if issubclass(territory_class, ResourceBasedTerritoryDataset)]
-        for territory_class in territory_classes:
-            dataset = Dataset.objects.get(id=territory_class.dataset_id)
-            for resource in dataset.resources:
-                if str(resource.id) != str(territory_class.resource_id):
-                    continue
-                filename = resource.url.split('/')[-1]
-                reference_path = references.path(filename)
-                log.info('Found reference: %s', reference_path)
-                if os.path.exists(reference_path):
-                    log.info('Reference already downloaded')
-                    continue
-                log.info('Downloading from: %s', resource.url)
-                with codecs.open(reference_path, 'w', encoding='utf8') as fd:
-                    r = requests.get(resource.url, stream=True)
-                    for chunk in r.iter_content(chunk_size=1024):
-                        fd.write(chunk.decode('latin-1'))  # TODO: detect?
-
+def migrate_zones_from_old_to_new_ids_in_datasets():
```

Comment:
Hum, it's a little bit long for a command name, why not `migrate_zone_ids` (and if you need to precise what's done, use the docstring which is displayed as help) ?